src/docs/development/data-and-backend/state-mgmt.md

```diff
@@ -6,32 +6,39 @@ description: How to structure an app to manage the state of the data flowing thr
 "How do I best manage the state of the UI controls in a medium-to-large
 Flutter app?" is a question we often hear.
 This is an complex topic with strong, and differing, opinions.
-Until we provide more docs/examples, find
-information at the following links, some of these are
+Until we provide more docs/examples, learn more at the
+following links, some of which have been
 contributed by the Flutter community:
 
-## State management
+## setState
```

Comment:
Capitalization?

Reply:
> No, it's a method name and we don't use codefonts in headers.

Right, but it should at least be something like:

> ## Using setState()

From the [guide](https://developers.google.com/style/code-in-text?hl=th#method-names): "Put an empty pair of parentheses after a method name to indicate that it's a method."

Reply:
> But that loses parallelism ...

Right, but we might not have a choice. IMHO, a heading of just "setState" isn't proper. How about:

> ## The setState() approach

@kwalrath, any opinion about this?